apps/emqx_authz/src/emqx_authz.erl

```diff
@@ -53,15 +53,25 @@
 
 -type(sources() :: [source()]).
 
-
+%% Initialize authz backend.
+%% Populate the passed configuration map with necessary data,
+%% like `ResourceID`s
 -callback(init(source()) -> source()).
 
+%% Get authz text description.
 -callback(description() -> string()).
 
+%% Destroy authz backend.
+%% Make cleanup of all allocated data.
+%% An authz backend will not be used after `destroy`.
 -callback(destroy(source()) -> ok).
 
+%% Check if a configuration map is valid for further
+%% authz backend initialization.
+%% The callback must deallocate all allocated resources.
```

Comment:
Could you clarify: does it need to free all resources that were allocated by `init(...)` or it should just leave the system in the same state as before?

Reply:
✅